src/igroupadmin.cpp

```diff
@@ -291,7 +291,8 @@ doCommand( char *cmdToken[] ) {
     }
 
     if ( strcmp( cmdToken[0], "mkuser" ) == 0 ) {
-        userAdmin( "mkuser", cmdToken[1], setScrambledPw( cmdToken[2] ),
+        char empty_string[] = "";
+        userAdmin( "mkuser", cmdToken[1], *cmdToken[2] ? setScrambledPw(cmdToken[2]) : empty_string,
```

Comment:
Can `cmdToken[2]` be null?
Is `cmdToken[2]` guaranteed to be null-terminated?

Reply:
For cmdToken in the client to be both non nullptr in value and null terminated depends on both: 
   - its assigned `argv` value to be null terminated, which they always are, assuming we don't scan past `argc` # of values on the cmdline (which we don't); backed up, of course, by the initialization of unassigned values to "" in the loop (in `main`, and in `getInput` for interactive mode) calls preceding the assignment.  
   - `getInput( )` to always guarantee said conditions, which a fairly thorough look-through seems to confirm.

However, I wouldn't argue against the possibility of improvement

Reply:
I see. The pointers of `cmdToken` always point to strings in `argv` therefore they are guaranteed to be null-terminated. That means checking the first byte for zero is acceptable as a check for empty string.

Good enough for me. Please mark as resolved when ready.